docs/source/release/v6.3.0/mantidworkbench.rst

```diff
@@ -17,6 +17,6 @@ New and Improved
 Bugfixes
 --------
 - Fixed arbitrary values not being accepted as the "Start Time" in StartLiveDataDialog.
-
+- Fixed plot bins not working on some data.
```

Comment:
LGTM and fixes the problem, could we explain what "some data" is for the future release editor who goes through the release notes in the future